packages/playwright/src/reporter.ts

```diff
@@ -159,289 +164,311 @@ export default class ReplayPlaywrightReporter implements Reporter {
   // TODO(PRO-667): this could be simplified to `${test.testId}-${test.repeatEachIndex}-${test.attempt}`
   // before doing that all recipients of `TestExecutionIdData` should be rechecked to see if such a change would be safe
   private _getTestExecutionId(test: TestExecutionIdData) {
-    return [
-      test.filePath,
-      test.projectName ?? "",
-      test.repeatEachIndex,
-      test.attempt,
-      ...test.source.scope,
-      test.source.title,
-    ].join("-");
+    return withSentrySync("GetTestExecutionId", () => {
```

Comment:
I don't think this makes sense> There's no function to wrap here, just an inline array.